examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,391 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f1e5a16f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "provider_calibrations = device.gate_calibrations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "959c5fe8",
+   "metadata": {},
+   "source": [
+    "Even for the same gate, different physical qubits and angles usually have a different pulse implementation. Because of this, a gate calibration may be associated with specific physical qubits and angles. In the code snippet below, we retrieve the gate calibration of the Rx gate for physical qubit `$0` and `angle=pi/2`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3248ed04",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OPENQASM 3.0;\n",
+      "cal {\n",
+      "    waveform wf_drag_gaussian_1 = drag_gaussian(24.0ns, 2.547965400864ns, 2.296242349815984e-10, 0.293772761206776, false);\n",
+      "    barrier $0;\n",
+      "    shift_frequency(q0_rf_frame, -321047.14178613486);\n",
+      "    play(q0_rf_frame, wf_drag_gaussian_1);\n",
+      "    shift_frequency(q0_rf_frame, 321047.14178613486);\n",
+      "    barrier $0;\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "pulse_sequence_rx_pi_2_q0 = provider_calibrations.pulse_sequences[(Gate.Rx(np.pi / 2), QubitSet(0))]\n",
+    "\n",
+    "print(pulse_sequence_rx_pi_2_q0.to_ir())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5579f6e",
+   "metadata": {},
+   "source": [
+    "Some gate calibrations are defined not with fixed values of angles, but rather with variables. In this case, these gate calibrations can be retrieved with angles of `FreeParameter` type. For example, the code snippet below retrieves the gate calibration of the Rz gate on the qubit `$1` with a variable angle `theta`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "81bb487f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OPENQASM 3.0;\n",
+      "cal {\n",
+      "    barrier $1;\n",
+      "    shift_phase(q1_rf_frame, -1.0*theta);\n",
+      "    shift_phase(q0_q1_xy_frame, 0.5*theta);\n",
+      "    shift_phase(q1_q2_xy_frame, 0.5*theta);\n",
+      "    shift_phase(q1_q16_xy_frame, 0.5*theta);\n",
+      "    barrier $1;\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "theta = FreeParameter(\"theta\")\n",
+    "pulse_sequence_rz_theta_q0 = provider_calibrations.pulse_sequences[(Gate.Rz(theta), QubitSet(1))]\n",
+    "\n",
+    "print(pulse_sequence_rz_theta_q0.to_ir())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d4d9641",
+   "metadata": {},
+   "source": [
+    "## Compose custom gate calibrations with AutoQASM\n",
+    "To customize a gate calibration, you need to \n",
```

Comment:
This doesn't cover everything in this section. I'd recommend either adding another subsection when you've reached the end of these points, so that the user knows these have been achieved, or else expand this list to cover the whole section